person.rb

```diff
@@ -0,0 +1,23 @@
+require 'securerandom'
+
+class Person
+  attr_accessor :name, :age
+  attr_reader :id
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = SecureRandom.uuid
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    is_of_age? && @parent_permission
+  end
```

Comment:
- [ ] Public method `can_use_services?` should return true if a person is of age or if they have permission from parents. Kindly, instead of using the && operator, I'll suggest you use | | operator (or operator). Example below:
```
  def can_use_services?
    is_of_age? || @parent_permission
  end
```

Reply:
Thanks for you observation 😎